terraform/azure/infra/variables.tf

```diff
@@ -20,6 +20,12 @@ variable "deployment_is_private" {
   default     = false
 }
 
+variable "use_web_application_firewall" {
+  description = "If true, we'll provision a web application firewall for increased security"
```

Comment:
Should we add a warning here and/or in the docs that removing this will decrease protection for the kubernetes workloads instead of adding it for increased security? Maybe [link](https://docs.microsoft.com/en-us/azure/architecture/example-scenario/aks-agic/aks-agic#:~:text=A%20Web%20Access,the%20WAF%20logs.) to the Azure docs where it says:

>A Web Access Firewall (WAF) Policy is associated to the Application Gateway at the root level and at the HTTP listener level, to protect tenant workloads from malicious attacks. The policy is configured in Prevention mode and uses [OWASP 3.1](https://owasp.org/www-project-application-security-verification-standard/) to block intrusions and attacks that are detected by rules. The attacker receives a "403 unauthorized access" exception, and the connection is closed. Prevention mode records these attacks in the WAF logs.